src/committee/orchestrator.rs

```diff
@@ -567,15 +574,26 @@ pub async fn run_server(
     let address = address.unwrap_or("127.0.0.1:6666");
     info!("- starting orchestrator at address http://{address}");
 
+    let mut compliance = Compliance::new();
+    // Orchestrator should sync the Sanction ist before doing anything else
+    compliance.sync().await.expect("sync sanction list");
+
+    // wrap in an Arc after the first sync so it can be used in multiple request contexts
+    let compliance: Arc<Compliance> = Arc::new(compliance);
+
     let member_status_state = Arc::new(RwLock::new(MemberStatusState::new(&committee_cfg).await));
     let mss_thread_copy = member_status_state.clone();
     tokio::spawn(async move { MemberStatusState::keepalive_thread(mss_thread_copy).await });
 
-    let ctx = Orchestrator {
+    let ctx = Orchestrator::new(
         pubkey_package,
         committee_cfg,
-        member_status: member_status_state,
-    };
+        member_status_state,
+        Arc::clone(&compliance),
+    );
+
+    // Sync sanction list in a parallel thread
+    compliance.start();
```

Comment:
don't you have to use the joinhandle returned?

Reply:
not if you don't want to wait for it to complete